styles/styles.css

```diff
@@ -350,46 +350,53 @@ div ul li {
 }
 
 .thumbnail {
-  display: flex;
-  flex-direction: column;
-  width: 164px;
-  height: 24px;
-  justify-content: center;
-  align-items: flex-start;
+  position: relative;
+  align-items: center;
 }
 
 .rectangle {
   position: absolute;
-  width: 18px;
-  height: 18px;
-  background-color: #ff6b00;
-  transform: rotate(-45deg);
-  left: 80px;
+  background-image: url(/img/Rectangle\ 551.png);
```

Comment:
[Optional]
- Your images are currently not showing😭. I believe the problem may be the path you specified to your image file. Please update the path to use a relative path by adding a `.` before the file path, e.g.`"./img/Rectangle\ 551.png"`🙏. Also make sure to wrap your URL in quotation marks. This ensures that your image files are found regardless of the platform.

- This affects all your images so please update all of them👍

**Note: The image may show on your computer, but it will not show anywhere else.**